python/tests/test_simulation.py

```diff
@@ -13,6 +13,9 @@
     unicode = str
 
 
+TIMESINKS_FOR_TEST = (mp.BoundarySteppingE, mp.MPBTime, mp.Connecting)
```

Comment:
Note that because this test does not involve eigenmode sources or mode decomposition, the time for `mp.MPBTime` will be zero across all processes. This could be something to verify as part of the test along with *non-zero* times for the other two categories `mp.BoundarySteppingE` and `mp.Connecting`.

Reply:
Updated.